src/LndClient/RPC.hs

```diff
@@ -307,3 +318,83 @@ subscribeInvoices env req invoiceHandler = rpc $ rpcArgs env
       Left e ->
         $(logTM) ErrorS $ logStr $ "failed to parse subscription invoice " <> e
       Right (ResultWrapper (i :: Invoice)) -> invoiceHandler i
+
+openChannel ::
+  (KatipContext m, MonadUnliftIO m) =>
+  LndEnv ->
+  OpenChannelRequest ->
+  m (LndResult (RPCResponse ChannelPoint))
+openChannel env req = rpc $ rpcArgs env
+  where
+    rpcArgs rpcEnv =
+      RpcArgs
+        { rpcEnv,
+          rpcMethod = POST,
+          rpcUrlPath = "/v1/channels",
+          rpcUrlQuery = [],
+          rpcReqBody = Just req,
+          rpcRetryAttempt = 0,
+          rpcSuccessCond = stdRpcCond,
+          rpcName = OpenChannel,
+          rpcSubHandler = Nothing
+        }
+
+getPeers ::
+  (KatipContext m, MonadUnliftIO m) =>
+  LndEnv ->
+  VoidRequest ->
```

Comment:
"VoidRequest" is literally void, it don't have any useful information inside, we can remove this argument

Reply:
Done